Source/SwiftLintCore/Rules/SuperfluousDisableCommandRule.swift

```diff
@@ -12,6 +12,16 @@ public struct SuperfluousDisableCommandRule: ConfigurationProviderRule, SourceKi
             in the disabled region. Use " - " if you wish to document a command.
             """,
         kind: .lint
```

Comment:
```suggestion
        kind: .lint,
```
This fails the build.